### PR TITLE
Move UI classes to .ui package

### PR DESCRIPTION
**Pull request by chanseokoh**
Moving `MenuContributionInitializer` and `GoogleLoginCommandHandler` to the `.ui` package.

### CODE REVIEW

**Comment by briandealwis** [comment]
One test failed due to a test timeout
```
Running com.google.cloud.tools.eclipse.appengine.localserver.server.LocalAppEnginePublishOperationTest
[ERROR] Timeout 600 s exceeded. Process was killed.
```
The other test actually crashed out the JVM!
```
Application Started: 6193
*** glibc detected *** /usr/lib/jvm/java-8-oracle/jre/bin/java: free(): invalid pointer: 0x0000000100021848 ***
======= Backtrace: =========
/lib/x86_64-linux-gnu/libc.so.6(+0x7da26)[0x7f07fb262a26]
/usr/lib/jvm/java-8-oracle/jre/lib/amd64/server/libjvm.so(+0x8de889)[0x7f07fab3d889]
/usr/lib/jvm/java-8-oracle/jre/lib/amd64/server/libjvm.so(+0x606011)[0x7f07fa865011]
/usr/lib/jvm/java-8-oracle/jre/lib/amd64/server/libjvm.so(+0x978050)[0x7f07fabd7050]
/usr/lib/jvm/java-8-oracle/jre/lib/amd64/server/libjvm.so(+0x978d76)[0x7f07fabd7d76]
/usr/lib/jvm/java-8-oracle/jre/lib/amd64/server/libjvm.so(+0xa812ef)[0x7f07face02ef]
/usr/lib/jvm/java-8-oracle/jre/lib/amd64/server/libjvm.so(+0xa81770)[0x7f07face0770]
/usr/lib/jvm/java-8-oracle/jre/lib/amd64/server/libjvm.so(+0x8e8158)[0x7f07fab47158]
/lib/x86_64-linux-gnu/libpthread.so.0(+0x7e9a)[0x7f07fb9c3e9a]
/lib/x86_64-linux-gnu/libc.so.6(clone+0x6d)[0x7f07fb2d836d]
======= Memory map: ========
00400000-00401000 r-xp 00000000 fc:09 125958682                          /usr/lib/jvm/java-8-oracle/jre/bin/java
[…]
```

File: plugins/com.google.cloud.tools.eclipse.login/src/com/google/cloud/tools/eclipse/login/ui/GoogleLoginCommandHandler.java [hunk 1]
```diff
@@ -14,20 +14,19 @@
  * limitations under the License.
  */
 
-package com.google.cloud.tools.eclipse.login;
+package com.google.cloud.tools.eclipse.login.ui;
 
-import com.google.cloud.tools.eclipse.login.ui.AccountsPanel;
+import com.google.cloud.tools.eclipse.login.IGoogleLoginService;
+import com.google.cloud.tools.eclipse.login.Messages;
```
Comment (briandealwis) [inline plugins/com.google.cloud.tools.eclipse.login/src/com/google/cloud/tools/eclipse/login/ui/GoogleLoginCommandHandler.java]: Would it make sense for .Messages to be moved to .ui too?

Comment (chanseokoh) [inline plugins/com.google.cloud.tools.eclipse.login/src/com/google/cloud/tools/eclipse/login/ui/GoogleLoginCommandHandler.java]: Hmm... all of our bundles have `Messages` at the top level, so I think I'll leave it there.

**Comment by chanseokoh** [comment]
> The other test actually crashed out the JVM!

Awesome!

**Pull request merged**